bayesian/v2/api_v2.py

```diff
@@ -223,10 +263,20 @@ def add_resource_no_matter_slashes(resource, route, endpoint=None, defaults=None
 
 
 add_resource_no_matter_slashes(ApiEndpoints, '')
+add_resource_no_matter_slashes(SystemVersion, '/system/version')
+
+# Component analyses routes
 add_resource_no_matter_slashes(ComponentAnalyses,
                                '/component-analyses/<ecosystem>/<package>/<version>',
-                               endpoint='get_component_analysis')
-add_resource_no_matter_slashes(SystemVersion, '/system/version')
+                               endpoint='get_component_analyses')
+
+# Stack analyses routes
+add_resource_no_matter_slashes(StackAnalysesPost,
```

Comment:
I don't think 2 separate routes are required. 
`def get(): pass`
`def post(): pass`
should do the this Job

Reply:
Comment taken.